tools/importer-rest-api-specs/components/terraform/resources/identification.go

```diff
@@ -235,12 +234,12 @@ func modelContainsDiscriminatedTypes(model models.SDKModel, data services.Resour
 		//if topLevelObjectDefinition.Type != resourcemanager.ReferenceApiObjectDefinitionType {
 		//	continue
 		//}
-		//nestedModel, isNestedModel := data.Schema.Models[*topLevelObjectDefinition.ReferenceName]
+		//nestedModel, isNestedModel := apiResource.Schema.Models[*topLevelObjectDefinition.ReferenceName]
```

Comment:
I can't leave a comment on line 232, but since this is a refactor and a bit of a clean up is there more context on that or an open issue we can link to?

Reply:
Good shout.

Taking a look into the history here, surprisingly it looks like [this has always been commented out](https://github.com/hashicorp/pandora/blame/5e86ea25f09456d4d9bd15659df32ec850260039/tools/importer-rest-api-specs/components/terraform/resources/identification.go#L233) - however checking this locally it appears that this should be fine to uncomment/fix this up - so I'll push a commit to do that.